### ISSUE TITLE
Final Review Request of seven (7) W3C VCWG Specifications > 2025-02-01

### ISSUE THREAD

**Issue by msporny** [w3c/i18n-request#255]
The W3C VCWG is requesting a *final* horizontal review of the following specifications, which your group has reviewed previous versions of over the past year or more.

We are requesting this before proceeding to the Proposed Recommendation phase based on a request by W3M to ensure that changes that have been made in the past six months are reviewed by your group since your last review. We are planning to enter the Proposed Recommendation phase for all seven (7) documents in Q1 2025 (roughly February, if possible).

 - [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-data-model-2.0/#revision-history)
   - [Implementation Report](https://w3c.github.io/vc-data-model-2.0-test-suite/)
 - [Verifiable Credential Data Integrity v1.0](https://www.w3.org/TR/vc-data-integrity/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-data-integrity/#revision-history)
   - Implementation Report (see ECDSA and EdDSA test suites, which test the Data Integrity base spec in each cryptosuite)
 - [ECDSA Cryptosuite v1.0](https://www.w3.org/TR/vc-di-ecdsa/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-di-ecdsa/#revision-history)
   - [Implementation Report](https://w3c.github.io/vc-di-ecdsa-test-suite/)
 - [EdDSA Cryptosuite v1.0](https://www.w3.org/TR/vc-di-eddsa/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-di-eddsa/#revision-history)
   - [Implementation Report](https://w3c.github.io/vc-di-eddsa-test-suite/)
 - [Securing Verifiable Credentials using JOSE and COSE](https://www.w3.org/TR/vc-jose-cose/)
   - [Changes since Candidate Recommendation](https://github.com/w3c/vc-jose-cose/commits/?since=2024-09-15&until=now)
   - [Implementation Report](https://w3c.github.io/vc-jose-cose-test-suite/)
 - [Bitstring Status List v1.0](https://www.w3.org/TR/vc-bitstring-status-list/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-bitstring-status-list/#revision-history)
   - [Implementation Report](https://w3c.github.io/vc-bitstring-status-list-test-suite/)
 - [Controlled Identifier Documents v1.0](https://www.w3.org/TR/controller-document/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/controller-document/#revision-history)
   - [Implementation Report](https://w3c.github.io/cid/implementations/1.0/)

  - Primary contacts:
      - Brent Zundel @brentzundel (Chair)
      - Ivan Herman @iherman (W3C Staff Contact)
      - Greg Bernstein @wind4greg (Editor, Data Integrity and cryptosuite specs)
      - Mike Jones @selfissued (Editor, VCDM v2.0 and VC JOSE COSE spec)
      - Gabe Cohen @decentralgabe (Editor, VC JOSE COSE spec)
      - Manu Sporny @msporny (Editor, VCDM v2.0, Data Integrity, Bitstring Status List specs)
      - Dmitri Zagidulin @dmitrizagidulin (Editor, Data Integrity and cryptosuite specs)
  - Organization/project driving the specification:
      - W3C Verifiable Credentials Working Group

Further details:

There are no major unresolved issues or opposition to these specifications that we know of at the present time.

**Comment by aphillips** [comment]
Thanks for this request. I have temporarily assigned an approximate due date of February 1. I may adjust this slightly once we discuss with the group and glance at the calendar. A cursory search of my dashboard shows no open issues on any of these specifications: I closed the tracker on bitstring's self-review, since your self-review is closed.

We will primarily review the changes reported above and may not repeat the full review of the documents.

**Comment by msporny** [comment]
> We will primarily review the changes reported above and may not repeat the full review of the documents.

Yes, that's the expectation (review changes only) as i18n has (IIRC) performed thorough reviews of each of these specifications in the past. I do not think we've made any i18n-related changes for any of the specs since the last time you reviewed them. This is just a "please make one last pass" request before we proceed to PR.
